### PR TITLE
Fixes up some loadout and clothes.

### PR DESCRIPTION
**Pull request by Belsima**
Fannypacks and flats now use colorable options rather than set color seleciton. Webbings and drop pouches use a selection, as do scarves. Added the knife harness to the webbing crate. Removed unused fannypack sprites.

### CODE REVIEW

**Comment by SilverTalismen** [comment]
We already have a colorable flats but I would prefer we keep the set color flats

**Comment by Belsima** [comment]
You can set it up to use any color the colored flats have- this just removes the 20 options of flats we have now that clutter up the loadout.

**Comment by Belsima** [comment]
Eh, I'll return it and fix the flats up a better way.

**Comment by Belsima** [comment]
...huh? I think I just fixed one thing after my test. Didn't realize I somehow made it not compile. One min.

**Comment by Anewbe** [comment]
That's an indentation error, iirc those compile but Travis just doesn't like them.

**Comment by Belsima** [comment]
Ahhh. Alright.

**Comment by Belsima** [comment]
Fixed.

File: code/datums/supplypacks/misc.dm [hunk 1]
```diff
@@ -64,7 +64,8 @@
 			/obj/item/clothing/accessory/storage/black_drop_pouches,
 			/obj/item/clothing/accessory/storage/brown_drop_pouches,
 			/obj/item/clothing/accessory/storage/white_drop_pouches,
-			/obj/item/clothing/accessory/storage/webbing
+			/obj/item/clothing/accessory/storage,
+			/obj/item/clothing/accessory/storage/knifeharness
```
Comment (Anewbe) [inline code/datums/supplypacks/misc.dm]: This is a bit out-of-scope.

Comment (Belsima) [reply code/datums/supplypacks/misc.dm]: My computer is in a cardboard box and will be for weems. Can't change it.

**Comment by Belsima** [comment]
Tried to fix on my phone. Let's see how badly that works.

**Pull request closed without merging**